example/dj/apps/app/resource.py

```diff
@@ -137,3 +145,23 @@ class UserWithFormResource(BaseModelResource):
     read_obj_permission = True
     update_obj_permission = True
     delete_obj_permission = True
+
+
+class IssueForm(RESTModelForm):
+
+    created_by = SingleRelatedField('created_by')
```

Comment:
Tohle tu takhle musí být? Myslel jsem že se to vygeneruje automaticky teda....

Reply:
rest, kde se to používá to má v testu vypnuté, proto to tu je. Je to čistě testovací záležitost